File: src/main/java/io/github/hotlava03/baclava4j/commands/basic/BasicCategory.java
package io.github.hotlava03.baclava4j.commands.basic;

import java.util.Arrays;
import java.util.List;

import io.github.hotlava03.baclava4j.commands.Category;
import io.github.hotlava03.baclava4j.commands.CategoryType;
import io.github.hotlava03.baclava4j.commands.Command;

public class BasicCategory implements Category {

    @Override
    public CategoryType getType() {
        return CategoryType.BASIC;
    }

    @Override
    public List<Command> getCommands() {
        return Arrays.asList(
            
        );
    }
    
}

File: src/main/java/io/github/hotlava03/baclava4j/commands/basic/BasicCategory.kt
package io.github.hotlava03.baclava4j.commands.basic

import io.github.hotlava03.baclava4j.commands.Category
import io.github.hotlava03.baclava4j.commands.CategoryType
import io.github.hotlava03.baclava4j.commands.Command

class BasicCategory : Category {
    override val type: CategoryType
        get() = CategoryType.BASIC

    override val commands: List<Command>
        get() = listOf()
}